### ISSUE TITLE
TXT record - State update failure due to sha verification issue

### ISSUE THREAD

**Issue by TristanBaoui** [akamai/terraform-provider-akamai#58]
### Terraform Version
Terraform v0.12.7
+ provider.akamai v0.1.3

### Affected Resource(s)
- akamai_dns_record

### Terraform Configuration Files
```hcl
resource "akamai_dns_record" "txt__dmarc_examples_com" {
	zone = "example.com"
	name = "_dmarc.example.com"
	recordtype = "TXT"
	active = true
	ttl = 300
	target = [
		"\"v=DMARC1;p=none;rua=mailto:dmarc@example.com\""
	]
	depends_on = [akamai_dns_zone.example_com]
}
```

### Debug Output
https://gist.github.com/TristanBaoui/9b7a2010fe27b2552c8887ab7afdc47b

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
TXT record should be created and terraform state updated to reflect it

### Actual Behavior
TXT record is created, but terraform state isn't updated due to the sha verification that fails after the record creation.

### Steps to Reproduce
1. `terraform apply`

### Code findings
In func GetRdata (https://github.com/akamai/AkamaiOPEN-edgegrid-golang/blob/a27cbfe442f75efbc159dd67f6d7bac3c0949960/configdns-v2/record_lookup.go) :
    line 165, double quote are removed from the strings (they are not before the other sha calculation)
    
SHA is different while the record was properly created (or even exists before the apply function)

**Comment by martinstibbe** [comment]
This will be fixed in 0.1.4 release coming up this week library has fix merged to master already

**Comment by martinstibbe** [comment]
https://releases.hashicorp.com/terraform-provider-akamai/0.1.4/ released please test and validate
